dropwizard-version-info-api/src/main/java/com/palantir/dropwizard/versioninfo/api/VersionInfoService.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.dropwizard.versioninfo.api;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * interface for obtaining the version info.
+ */
+@Path("")
+public interface VersionInfoService {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("version")
+    String getVersion();
```

Comment:
don't want to add it to the api project. I think there is also an issue with having that annotation for feign